src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusRunContext.java

```diff
@@ -31,7 +33,11 @@ protected static QuarkusRunContext getContext(AnActionEvent e) {
         Project project = PlatformDataKeys.PROJECT.getData(e.getDataContext());
         RunContentManager contentManager = RunContentManager.getInstance(project);
         RunContentDescriptor selectedContent = contentManager.getSelectedContent();
-        return (QuarkusRunContext) selectedContent.getComponent().getClientProperty(QuarkusConstants.QUARKUS_RUN_CONTEXT_KEY);
+        JComponent component = selectedContent.getComponent();
+        if (component == null) {
```

Comment:
Could be rewritten as:
`return component == null?null:(QuarkusRunContext) component.getClientProperty(QuarkusConstants.QUARKUS_RUN_CONTEXT_KEY)`

Reply:
Sure, I just wanted to make it easier to debug ;)